io/net/netperf_test.py

```diff
@@ -123,12 +123,23 @@ def setUp(self):
             self.cancel("unable to copy the netperf into peer machine")
         self.netperf_dir_peer = "/tmp/%s" % self.version
         self.netperf_dir = os.path.join(self.netperf, self.version)
-        cmd = "unzip /tmp/%s -d /tmp;cd %s;./configure --build=powerpc64le;make" % (
-            os.path.basename(tarball), self.netperf_dir_peer)
-        output = self.session.cmd(cmd)
+        cmd1 = "unzip /tmp/%s -d /tmp;cd %s" % (os.path.basename(tarball), self.netperf_dir_peer)
+        output = self.session.cmd(cmd1)
         if not output.exit_status == 0:
             self.fail("test failed because command failed in peer machine")
         os.chdir(self.netperf_dir)
+        patch_file = self.params.get('patch', default='nettest_omni.patch')
```

Comment:
you do not need to get the patchfile parameter,  this should be placed in data folder beside yamls and the same path should be used here